src/lib/chat/matrix-client.ts

```diff
@@ -508,7 +508,7 @@ export class MatrixClient implements IChatClient {
       firstName: user?.displayName,
       lastName: '',
       profileId: '',
-      isOnline: user?.presence === 'online',
+      isOnline: false,
```

Comment:
No reason to provide an incorrect value here, I don't think? Just don't return this field.

Reply:
If removed we will get a type error due to the type being `UserModel`. Should I change the method type?

Reply:
Oh, it's required? I guess...yeah, just leave this for now.